async function main() {
  // NOP
}

main().catch((e) => { console.error(e.stack || e); });
